frontend/src/components/widgets/BaseWidget/styled-components.ts

```diff
@@ -12,6 +12,12 @@ export const StyledWidgetLabelHelp = styled.div(() => ({
   right: 0,
 }))
 
+export const StyledWidgetLabelHelpTopLeft = styled.div(() => ({
+  position: "absolute",
+  left: "-30px",
+  top: "1px",
```

Comment:
The vertical alignment of the tooltip could perhaps use some tweaking? In my head, aligning so the top or the center of the (?) is aligned with the top of the image/chart would look better, roughly -8px or -13px.

Not sure how final the CSS here is intended to be in this PR though (eg if Abhi was planning on doing a design tweak later)